modules/veteran/app/models/veteran/service/organization.rb

```diff
@@ -5,7 +5,8 @@ module Veteran
   module Service
     class Organization < ApplicationRecord
       FUZZY_SEARCH_THRESHOLD = 0.3 # pg_search's default
```

Comment:
Can we move the following `organization` and `representative` model constants into `Veteran::Service::Constants`? They values for them are the same in both models.

```
      FUZZY_SEARCH_THRESHOLD = 0.3 # pg_search's default
      DEFAULT_MAX_MILES = 50
      DEFAULT_MAX_DISTANCE = DEFAULT_MAX_MILES * Constants::METERS_PER_MILE
```

Reply:
Done!

Reply:
Awesome!